corehq/apps/reports/cache.py

```diff
@@ -1,95 +1,60 @@
-from django.conf import settings
```

Comment:
Just noticed how bad this diff is. Whoever reviews, I'd just review the whole file (or at least review the diff in split mode) because it's a complete rewrite of the original file. (There are literally no substantive lines shared between the new and old file in this diff!)